winui/DataGrid/Row-Drag-and-Drop.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: post
+title: Row drag and drop in WinUI DataGrid control | Syncfusion
+description: Learn here all about row drag and drop support in Syncfusion WinUI DataGrid (SfDataGrid) control and more.
+platform: winui
+control: SfDataGrid
+documentation: ug
+---
+
+# Row drag and drop in WinUI DataGrid (SfDataGrid)
+
+[WinUI DataGrid](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.DataGrid.SfDataGrid.html) allows drag and drop the rows by setting the `AllowDraggingRows` and [AllowDrop](https://docs.microsoft.com/en-us/dotnet/api/system.windows.uielement.allowdrop) property as `true`. You can also drag and drop the rows between datagrid and other controls such as `ListView` , `SfTreeView` and `SfTreeGrid`. 
+
+{% tabs %}
+{% highlight xaml %}
+<syncfusion:SfDataGrid x:Name="sfDataGrid"
+                     AllowDraggingRows="True"
+                     AllowDrop="True"               
+                     AutoGenerateColumns="True"
+                     ItemsSource="{Binding Orders}" />
+{% endhighlight %}
+
+{% highlight c# %}
+this.sfDataGrid.AllowDraggingRows = true;
+this.sfDataGrid.AllowDrop = true;
+{% endhighlight %}
+{% endtabs %}
+
+While dropping, the dragged rows can be added above or below to the target record based on its drop position
```

Comment:
While dropping, the dragged rows can be added above or below to the target record based on their drop position.

Keep full stop at the end.